tools/ArduinoCsCompiler/Frontend/Frontend.csproj

```diff
@@ -9,20 +9,38 @@
     <AssemblyName>acs</AssemblyName>
     <RootNamespace>ArduinoCsCompiler</RootNamespace>
     <Product>Arduino CS Compiler</Product>
-    <Description>A translation compiler that retargets C# applications so they can run on a microcontroller using the unnamed firmware.</Description>
+    <Description>A translation compiler that retargets C# applications so they can run on a microcontroller using a special firmware. See Readme for further information.</Description>
     <Authors>The .NET Foundation community</Authors>
     <NeutralLanguage>en</NeutralLanguage>
+    <IsPackable>True</IsPackable>
+    <PackAsTool>True</PackAsTool>
+    <ToolCommandName>dotnet-acs</ToolCommandName>
+    <PackageOutputPath>..\..\..\artifacts\packages\$(Configuration)\Shipping</PackageOutputPath>
+    <Version>1.0.2</Version>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="*.cs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\..\src\devices\Arduino\samples\ApiChecker\RgbLedTest.cs" Link="RgbLedTest.cs" />
+    <Compile Include="..\..\..\src\devices\Arduino\samples\ApiChecker\TestCases.cs" Link="TestCases.cs" />
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\devices\Arduino\Arduino.csproj" />
-    <ProjectReference Include="..\..\..\src\devices\Arduino\samples\ApiChecker\Arduino.sample.csproj" />
     <ProjectReference Include="..\ArduinoCsCompiler.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="$(Configuration)=='Debug'">
```

Comment:
Can you add a comment here to explain the distinction of why Debug and Release are different?